adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenSilentHandlerTest.java

```diff
@@ -889,4 +900,8 @@ public boolean isConnectionAvailable() {
         }
 
     }
+
+    private Context getContext() {
+        return InstrumentationRegistry.getContext();
```

Comment:
Please be consistent. if you are adding this method, do it everywhere or a common place or I don't see any harm in leaving InstrumentationRegistry.getContext() in place